host/4/bullseye/amd64/node/node18/node18-composite.template

```diff
@@ -0,0 +1,54 @@
+FROM mcr.microsoft.com/dotnet/runtime-deps:6.0
+ARG HOST_VERSION
+
+COPY --from=runtime-image [ "/FuncExtensionBundles", "/FuncExtensionBundles" ]
+COPY sshd_config /etc/ssh/
+COPY start.sh /azure-functions-host/
+COPY --from=runtime-image [ "/azure-functions-host", "/azure-functions-host" ]
+COPY --from=runtime-image [ "/workers/node", "/azure-functions-host/workers/node" ]
```

Comment:
Verifying that Layer sharing is good here. There is only one set of node worker files right? The code just depends on which Node Lang Version is available? If there is more than one node worker defined in Host let me know and I can work on optimizing these images.

Reply:
Do you have an example of a language that has more than one worker in the host? Are you talking about PowerShell how they have a 7.0 and a 7.2 worker? If so, we are _not_ like PowerShell.

We use the same worker in v4 for all supported node versions (v14, v16 & now v18). We have a different worker in v3 of the host, but I don't think that's what you're asking about - and it's still just one worker used for all of v3.

Reply:
Another example is Python. They produce different flavors of worker per language version and I copy over just one of the workers like this https://github.com/Azure/azure-functions-docker/blob/dev/host/3.0/buster/amd64/python/python36/python36-composite.template#L69